python/daqconf/apps/trigger_gen.py

```diff
@@ -388,11 +404,17 @@ def get_trigger_app(
     mgraph = ModuleGraph(modules)
 
     if USE_HSI_INPUT:
-        mgraph.connect_modules("ttcm.output",         "tctee_ttcm.input",             "TriggerCandidate", "ttcm_input", size_hint=1000)
-        mgraph.connect_modules("tctee_ttcm.output1",  "mlt.trigger_candidate_input", "TriggerCandidate","tcs_to_mlt", size_hint=1000)
-        mgraph.connect_modules("tctee_ttcm.output2",  "tc_buf.tc_source",             "TriggerCandidate","tcs_to_buf", size_hint=1000)
+        mgraph.connect_modules("ttcm.output",              "tctee_ttcm.input",            "TriggerCandidate", "ttcm_input", size_hint=1000)
+        mgraph.connect_modules("tctee_ttcm.output1",       "mlt.trigger_candidate_input", "TriggerCandidate", "tcs_to_mlt", size_hint=1000)
+        mgraph.connect_modules("tctee_ttcm.output2",       "tc_buf.tc_source",            "TriggerCandidate", "tcs_to_buf", size_hint=1000)
         mgraph.add_endpoint("hsievents", "ttcm.hsi_input", "HSIEvent", Direction.IN)
 
+    if USE_CTB_INPUT:
+        mgraph.connect_modules("ctbtcm.output",              "tctee_ctbtcm.input",          "TriggerCandidate", "ctbtcm_input", size_hint=1000)
+        mgraph.connect_modules("tctee_ctbtcm.output1",       "mlt.trigger_candidate_input", "TriggerCandidate", "tcs_to_mlt",   size_hint=1000)
+        mgraph.connect_modules("tctee_ctbtcm.output2",       "tc_buf.tc_source",            "TriggerCandidate", "tcs_to_buf",   size_hint=1000)
+        mgraph.add_endpoint("hsievents", "ctbtcm.hsi_input", "HSIEvent", Direction.IN)
```

Comment:
I think this will need to be `ctbhsievents`, although the corresponding endpoint in ctb (here: https://github.com/DUNE-DAQ/ctbmodules/blob/f4d5ea7119a17399ff6b9f526309393d00ee891b/python/ctbmodules/apps/ctb_hsi_gen.py#L160) will need to change for that too. @strilov would you like us to change it on the ctb site (do we need to be part of the correct github team?) or someone from CTB will do this?

Reply:
@ArturSztuc I can make the change in the CTBModule. L160 that you pointed to I can change from `hsievents` to ctbhsievents`.

Reply:
Thank you @jsensenig , looks like @strilov has already made a branch for this here: https://github.com/DUNE-DAQ/ctbmodules/compare/production/v4...feature/individual_hsi_tc_makers
It will probably need to be merged together with this one

Reply:
Ah I missed that, it looks like it's taken care of then. We'll just need to merge in his changes.